addons/web/static/lib/owl/owl.js

```diff
@@ -3200,7 +3200,13 @@
             this.translateFn = config.translateFn;
             this.translatableAttributes = config.translatableAttributes;
             if (config.templates) {
-                this.addTemplates(config.templates);
+                if (config.templates instanceof Document || config.templates instanceof String) {

```

Comment:
Should be move to owl repo of course

Reply:
Do we need diff in owl at all? We can just call addTemplate in a loop from outside owl right?

Reply:
Yes, but everytime we create a new app we would have to make this loop.
But we could maybe make an helper to create an App

Reply:
I created one not long ago: mountComponent